www/modules/migrate/controllers/outlook.client.controller.es6

```diff
@@ -1,46 +1,58 @@
 const DATE_FORMAT = 'yyyy-MM-dd';
 
```

Comment:
Is better to store const in https://docs.angularjs.org/api/auto/service/$provide#constant